Apps/Playground/Android/app/src/main/cpp/BabylonNativeJNI.cpp

```diff
@@ -55,7 +56,7 @@ extern "C"
         g_chromeDevTools.reset();
         g_nativeXr.reset();
         g_scriptLoader.reset();
-        g_inputBuffer.reset();
+        g_nativeInput = {};
```

Comment:
Is there a threading issue here? g_nativeInput is being assigned/constructed on the JS thread, but being reset on the main thread. Not sure if there are lurking issues. Granted, this is what we are doing with the other objects also, but we don't check and/or use these objects on the main thread.

Reply:
I think there is potentially an issue here. I think with a different change we should rework `NativeInput` to be structured more like `NativeXr` and `NativeCamera`, where there is an impl object that has a native "projection" and a JS "projection" and the impl is ref counted. I think that is a safer approach, but I think that should be fixed separately.

Reply:
Unless you think that I should address this in this PR, I think that it'd be best to address a rework in a different follow-up PR.

Reply:
Okay with me.